sdk/src/types/block/error.rs

```diff
@@ -364,9 +365,10 @@ impl fmt::Display for Error {
             Self::UnallowedFeature { index, kind } => {
                 write!(f, "unallowed feature at index {index} with kind {kind}")
             }
-            Self::UnallowedUnlockCondition { index, kind } => {
+            Self::DisallowedUnlockCondition { index, kind } => {
                 write!(f, "unallowed unlock condition at index {index} with kind {kind}")
```

Comment:
Should also be renamed here then?
```suggestion
                write!(f, "disallowed unlock condition at index {index} with kind {kind}")
```